src/main/java/com/checkout/marketplace/MarketplaceAccountHolder.java

```diff
@@ -1,21 +1,26 @@
 package com.checkout.marketplace;
 
+import com.checkout.common.Address;
 import com.checkout.common.CountryCode;
+import com.checkout.common.Phone;
 import com.checkout.common.four.AccountHolderType;
-import com.checkout.common.four.AccountHolder;
 import com.checkout.common.four.SenderIdentification;
 import com.google.gson.annotations.SerializedName;
 import lombok.Data;
-import lombok.EqualsAndHashCode;
 import lombok.ToString;
 
 @Data
-@EqualsAndHashCode(callSuper = true)
 @ToString(callSuper = true)
-public final class MarketplaceAccountHolder extends AccountHolder {
+public final class MarketplaceAccountHolder {
```

Comment:
Is it worth it to have this class? It's basically the same as `common.four.AccountHolder`